docs/workflows.md

```diff
@@ -1 +1,94 @@
-TODO
\ No newline at end of file
+# Workflows
+
+The purpose of this section of the Kn documentation is to list what we believe are common workflows or use-cases for the Knative CLI. Obviously, as we release the CLI in the wild and get users in production, we might discover novel workflows. As we do we will improve this document.
+
+## Basic (get your feet wet)
+
+In this basic worflow we show the CRUD (create, read, update, delete) operations on a service. We use a well known [simple Hello World service](https://github.com/knative/docs/tree/master/docs/serving/samples/hello-world/helloworld-go) that reads the environment variable `TARGET` and prints it as output.
+
+1. Create a service from image in `default` namespace
+
+```bash
+$ kn service create hello --image gcr.io/knative-samples/helloworld-go --env TARGET=Knative
+```
+
+1. Curl service endpoint
+
+```bash
+curl '-sS' '-H' 'Host: hello.default.example.com' 'http://xxx.xx.xxx.xx:80'
+Hello Knative!
+```
+
+Where `http://xxx.xx.xxx.xx:80` is your Knative installation ingress.
+
+1. List service
+
+```bash
+$ kn service list
```

Comment:
I think we agreed that this will be `knative service get` (same semantics as in kubectl)

Reply:
OK let me see if in latest code I can do this. Otherwise, I'll omit until ready.

Reply:
@rhuss the service get issue #48 is implemented in #90 but seems that its on hold waiting on feedback and perhaps some code changes. 

I added a comment to include a section for `* Get service` when that's ready.